service-spec/schema-validator/pom.xml

```diff
@@ -22,5 +22,11 @@
 			<artifactId>json-schema-validator</artifactId>
 			<version>${json.schema.validator.version}</version>
 		</dependency>
-	</dependencies>
+		<dependency>
+			<groupId>io.hyscale</groupId>
+			<artifactId>service-spec-commons</artifactId>
+			<version>${project.version}</version>
+			<scope>compile</scope>
+		</dependency>
+    </dependencies>
```

Comment:
Why is schema validator dependent on spec-commons ?

Reply:
yes for enum of yaml and json